packages/netsuite-adapter/src/client/sdf_client.ts

```diff
@@ -795,6 +798,20 @@ export default class SdfClient {
       }
     }
 
+    const filesToSize = async (filePaths: string[], fileCabinetDirPath: string):
+      Promise<{ [path: string]: number }> => {
+      const normalizedPath = (filePath: string): string => {
+        const filePathParts = filePath.split(FILE_CABINET_PATH_SEPARATOR)
+        return osPath.join(fileCabinetDirPath, ...filePathParts)
+      }
+      return Object.assign({}, ...await withLimitedConcurrency(
+        filePaths.map(filePath => async () => (
+          { [filePath]: (await stat(normalizedPath(filePath))).size }
+        )),
+        READ_CONCURRENCY
+      ))
```

Comment:
Better to avoid `...` in potentially long arrays/objects.
```suggestion
      return Object.fromEntries(await withLimitedConcurrency(
        filePaths.map(filePath => async () => [filePath, await stat(normalizedPath(filePath))).size])),
        READ_CONCURRENCY
      ))
```

Reply:
I started with that, but I didn't manage to set up the types correctly. I'll try again